packages/contracts/contracts/protocol/lendingpool/LendingPoolConfigurator.sol

```diff
@@ -111,6 +108,62 @@ contract LendingPoolConfigurator is
     }
 
 
+    /**
+     * @dev Verifies tranche for more privileges
+     * @param trancheId the tranche that is verified
+     **/
+    function verifyTranche(
+        uint64 trancheId
+    ) external onlyGlobalAdmin {
+        require(trancheId<totalTranches, Errors.INVALID_TRANCHE);
+        require(!pool.getTrancheParams(trancheId).verified, Errors.ALREADY_VERIFIED);
+        // TODO: enforce possible staking of VMEX
+
+        // loop through all reserves in this tranche, and set the local values of LTV, threshold, bonus, and borrow factor
+        address[] memory reserves = pool.getReservesList(trancheId);
+        uint cachedLength = reserves.length;
+        for(uint i = 0;i<cachedLength; ++i) { //initialize with initial global asset params
+            (
+                uint64 ltv,
+                uint64 liquidationThreshold,
+                uint64 liquidationBonus,
+                uint64 borrowFactor
+            ) = assetMappings.getDefaultCollateralParams(reserves[i]); //it doesn't matter what the tranche and user address is since it will get params from global params
+            pool.setCollateralParams(
+                reserves[i], 
+                trancheId, 
+                ltv, 
+                liquidationThreshold, 
+                liquidationBonus, 
+                borrowFactor
+            );
+
+            _setReserveInterestRateStrategyAddress(
+                reserves[i], 
+                trancheId, 
+                assetMappings.getDefaultInterestRateStrategyAddress(reserves[i])
+            );
+        }
+
+        pool.setTrancheAdminVerified(trancheId, true);
+
+        emit TrancheVerified(trancheId);
+    }
+
+
+    /**
+     * @dev Unverifies tranche to revoke privileges, unstake, and return to global parameters
+     * @param trancheId the tranche that is verified
+     **/
+    function unverifyTranche(
+        uint64 trancheId
+    ) external onlyVerifiedTrancheAdmin(trancheId) {
+        require(trancheId<totalTranches, Errors.INVALID_TRANCHE);
+        // TODO: enforce unstaking of VMEX
+        pool.setTrancheAdminVerified(trancheId, false);
+
+        emit TrancheUnverified(trancheId);
```

Comment:
The setTrancheAdminVerified function already emits ConfigurationAdminVerifiedUpdated(trancheId, verified)

This event emission may not be necessary

Reply:
OK removed